src/stores/PersonenkontextStore.ts

```diff
@@ -275,7 +276,7 @@ export const usePersonenkontextStore: StoreDefinition<
       this.loading = true;
       try {
         const updateParams: DbiamUpdatePersonenkontexteBodyParams = {
-          lastModified: new Date().toISOString(),
+          lastModified: this.personenuebersicht?.lastModifiedZuordnungen ?? new Date().toISOString(),
```

Comment:
Das FE kann sich dieses Datum nicht ausdenken. Wenn das BE das nicht mitschickt, dann muss das FE es leer zurück schicken.